<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="rahulshettyacademy.TestComponent.Listeners"/>
  </listeners>
  <test thread-count="5" name="Submit order Test">
    <classes>
      <class name="rahulshettyacademy.Tests.SubmitOrderTest"/>
    </classes>
  </test> <!-- Submit order Test -->
  <test thread-count="5" name="Error vaalidation Test">
    <classes>
      <class name="rahulshettyacademy.Tests.ErrorValidationTest"/>
    </classes>
  </test> <!-- Error vaalidation Test -->
  <test parallel="tests" name="Submit order Test(failed)">
    <classes>
      <class name="rahulshettyacademy.Tests.ErrorValidationTest">
        <methods>
          <include name="launchApplication"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- rahulshettyacademy.Tests.ErrorValidationTest -->
      <class name="rahulshettyacademy.Tests.SubmitOrderTest">
        <methods>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="submitOrder" invocation-numbers="1"/>
          <include name="OrderHistory"/>
        </methods>
      </class> <!-- rahulshettyacademy.Tests.SubmitOrderTest -->
    </classes>
  </test> <!-- Submit order Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
